source/brailleDisplayDrivers/braillePen12.py

```diff
@@ -0,0 +1,182 @@
+#brailleDisplayDrivers/braillePen12.py
+#A part of NonVisual Desktop Access (NVDA)
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+#Copyright (C) 2014 Harpo Sp. z o. o. <http://www.harpo.com.pl>
+
+import braille
+import queueHandler
+from logHandler import log
+from ctypes import *
+from ctypes.wintypes import *
+import time
+import config
+import inputCore
+import hwPortUtils
+import brailleInput
+import wx
```

Comment:
nit: These imports aren't used anywhere: queueHandler, log, time, config, wx. Please remove them. I think you also don't need ctypes.wintypes, but I'm not certain about that.